vllm/entrypoints/openai/serving_engine.py

```diff
@@ -84,6 +84,38 @@ async def show_available_models(self) -> ModelList:
         model_cards.extend(lora_cards)
         return ModelList(data=model_cards)
 
+
+    async def add_lora(self, lora: LoRA) -> ModelCard:
+        last_lora_request: LoRARequest = self.lora_requests.get(self.lora_requests.size() - 1)
+        lora_idx = 1
+        if last_lora_request is not None:
+            lora_idx = last_lora_request.lora_int_id + 1
+
+        self.lora_requests.append(LoRARequest(
+            lora_name=lora.name,
+            lora_int_id=lora_idx,
+            lora_local_path=lora.local_path,
+        ))
```

Comment:
let's have an increment only counter for LoRA ids instead?

also, is it possible to check if the lora has already been added before? we can use the local_path

Reply:
sorry for the delay, I will be active on this now; I will resume work here.